common/Makefile.am

```diff
@@ -30,6 +30,7 @@ libyami_common_ldflags = \
 	$(LIBVA_LIBS) \
 	$(LIBVA_DRM_LIBS) \
 	-lpthread \
+	-std=c++11 \
```

Comment:
std::strtol does not requires c++11.  Only std::strtoll does.

http://en.cppreference.com/w/cpp/string/byte/strtol